ReactiveCocoa/Swift/UnidirectionalBinding.swift

```diff
@@ -0,0 +1,315 @@
+import enum Result.NoError
+
+infix operator <~ {
+	associativity right
+
+	// Binds tighter than assignment but looser than everything else
+	precedence 93
+}
+
+public protocol BindingConsumer: class {
```

Comment:
Maybe `BindingDestination` would be a better name then, since the documentation already refers to _sources_ and _destinations_?